src/audio/kpb.c

```diff
@@ -430,6 +430,7 @@ static int kpb_prepare(struct comp_dev *dev)
 	struct list_item *blist;
 	struct comp_buffer *sink;
 	size_t hb_size_req = KPB_MAX_BUFFER_SIZE(kpb->config.sampling_width);
+	size_t hb_drain_size = KPB_MAX_DRAINING_SIZE(kpb->config.sampling_width);
```

Comment:
on a side note, what does 'sampling_width" mean? this is horribly confusing. Please consider renaming this field if it means 'drain-buffer-size'